tests/Unit/Controllers/HomeControllerTest.php

```
@@ -538,6 +538,70 @@ public function testLinkJiraTrimsKeyWhitespace(): void
+        $ctrl = new HomeController(
+            $this->makePostRequest(['name' => 'New', 'visibility' => 'everyone']),
+            $this->response,
+            $this->auth,
+            $this->db,
+            $this->config
+        );
+        $ctrl->editProject('42');
+
+        $this->assertStringContainsString('/app/home', $this->response->redirectedTo ?? '');
+        $this->assertStringContainsString('could not be updated', $_SESSION['flash_error'] ?? '');
+    }
```

PHPMD (2.15.0)

[error] testEditProjectSetsFlashErrorOnDbException accesses the super-global variable $_SESSION. (undefined)

(Superglobals)